lib/suspenders/app_builder.rb

```diff
@@ -33,7 +33,7 @@ def provide_setup_script
     end
 
     def provide_dev_prime_task
-      copy_file 'development_seeds.rb', 'lib/tasks/development_seeds.rake'
+      copy_file 'development_data.rb', 'lib/tasks/development_data.rake'
```

Comment:
> What about `dev.rake`, so it matches its namespace?

Agreed! It's easier to find rake tasks if they're defined according to convention.